misk/src/main/kotlin/misk/web/metadata/webaction/WebActionMetadata.kt

```diff
@@ -46,10 +52,19 @@ data class WebActionMetadata(
   ) : this(
     name = name,
     function = function.toString(),
+    packageName = packageName(function.toString()),
```

Comment:
The `function` string is created from a `KFunction` :
https://github.com/cashapp/misk/blob/9593078083385e235dda816af539bd48bae616c3/misk/src/main/kotlin/misk/Action.kt#L14

So we might be able to get the package name easier by calling [javaMethod](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.reflect.jvm/java-method.html).[getDeclaringClass()](https://docs.oracle.com/javase/8/docs/api/java/lang/reflect/Method.html#getDeclaringClass--).[getPackage()](https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html#getPackage--) and adding that as another string parameter to `WebActionMetadata`.

Reply:
Thanks Tim! Addressed in: https://github.com/cashapp/misk/pull/1850/commits/e5bfa6f5c5c70522b57ded6a029e42636d41b2d0